zuul.d/project.yaml

```diff
@@ -0,0 +1,7 @@
+---
+- project:
+    name: openstack-k8s-operators/ovn-operator
```

Comment:
This will need to be updated

Reply:
Could be causing a duplicate with https://github.com/openstack-k8s-operators/ovn-operator/blob/0bb142a7d5cdd51e1c9c8a127e837bb522619fda/zuul.d/project.yaml#L3

Reply:
Nice, I was able to push the commit to test the fix into your PR. I Zuul is executing now :)